tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compileOnSave": false,
+  "compilerOptions": {
+    "baseUrl": "./",
```

Comment:
I prefer we always use relative paths to make it super clear what an import is referring to. What would you think of not defining `baseUrl` here? Even if you would want to use it in packages, I think it would be confusing that in a package that extends this `tsconfig` the baseUrl would be relative to the root of this repo.